### PR TITLE
Implement database migrations @gear-js/data-storage

### PR DESCRIPTION
**Pull request by Almazatun**

### CODE REVIEW

File: website/data-storage/package.json [hunk 1]
```diff
@@ -65,9 +69,9 @@
     "supertest": "6.2.3",
     "ts-jest": "27.1.5",
     "ts-loader": "8.4.0",
-    "ts-node": "9.1.1",
+    "ts-node": "^10.8.1",
```
Comment (osipov-mit) [inline website/data-storage/package.json:72]: pls remove `^` because renovate-bot don't understand this

Comment (Almazatun) [reply website/data-storage/package.json:72]: Alright

File: website/data-storage/README.md [hunk 2]
```diff
@@ -1 +1,14 @@
 # @gear-js/data-storage
+
+### Database migration
+
+```bash
+# generate migration
+$ yarn migration:generate src/database/migrations/<MIGRATION_NAME>
```
Comment (osipov-mit) [inline website/data-storage/README.md:7]: Is there any migration naming convention?

Comment (Almazatun) [reply website/data-storage/README.md:7]: Simple answer the `question`, yes.

Depend on the convention between developers in `team`. 
Usually, migration has two basic parts:
1. migration number or id (That part will generate automatically with TypeORM).
2. the second part is the name of migration. The name should tell us what should do a new migration. Some people write the second part with `camelcase` or `snakecase`.

**Review by Pinkdesu — APPROVED** [review]

**Pull request merged**